src/services/helper/sessions.js

```diff
@@ -44,6 +45,9 @@ module.exports = class SessionsHelper {
 				bodyData['endDateUtc'] = moment.unix(bodyData.endDate).utc().format(common.UTC_DATE_TIME_FORMAT)
 			}
 
+			const mentorName = await UserProfileHelper.details('', loggedInUserId)
```

Comment:
@ankitpws this is not required, and don't store mentor name in DB.

In response only we need to send the result mentor name